php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2029,25 +2056,24 @@ class NDB_BVL_Instrument extends NDB_Page
 
     /**
      * Calculates the candidate's age at the time of this instrument.
-     * If Date_taken was based in, it will calculate the age as of
-     * the date passed as a parameter (this is used, for instance, in
+     * If Date_taken is passed, this function  will calculate the age as of
+     * the value of that argument. This is used, for instance, in
      * _saveValues when we want to calculate the age based on the new
-     * value and not based on the age saved in the database), otherwise
-     * it will calculate based on the date saved into the Date_taken
-     * field from the database.
+     * value and not based on the age saved in the database. Otherwise
+     * the date will be calculated based on the value of the Date_taken
+     * field in the database.
      *
-     * @param array $Date_taken (optional) The date that the age is to
+     * @param array $date (optional) The date that the age is to
      *                          be calculated based on.
      *
      * @return array The age of the candidate in an array containing years, months
      *         and days.
      */
-    function getCandidateAge($Date_taken = false)
```

Comment:
The bool case in this function was removed.

Reply:
@johnsaigle the parameter here should be a string not array